e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts

```diff
@@ -6,11 +6,15 @@ interface Project {
 }
 
 describeIfIAMV2p1('project management', () => {
+  // we increase the default delay to mimic the average human's typing speed
+  // only need this for input values upon which later test assertions depend
+  // ref: https://github.com/cypress-io/cypress/issues/534
+  const typeDelay = 50;
```

Comment:
tbh I'm not sure `typeDelay` is worth keeping, it hasn't really decreased flakiness in buildkite as far as I can tell. Instead we could add some `should` assertions to check that the form and save button are loaded